Assets/MixedRealityToolkit/Inspectors/Profiles/BaseMixedRealityProfileInspector.cs

```diff
@@ -362,22 +322,14 @@ private static bool IsConcreteProfileType(String profileTypeName)
             return profileTypeName != BaseMixedRealityProfileClassName;
         }
 
-        private static void CreateStyles()
+        /// <summary>
+        /// Checks if the profile is locked and displays a warning.
```

Comment:
>displays a warning. [](start = 48, length = 19)

The implementation down below doesn't seem to display a warning - it looks like both calls down below are just property getters. Maybe this part of the comment is stale and can be deleted?